messages/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+
+class Message(models.Model):
+    user = models.ManyToManyField(User, blank=True)
+    date = models.DateTimeField(default=timezone.now)
+    text = models.TextField()
+    # Message.id field auto-generated by Django, will be used down the road but no need to define it
+    category = models.TextField(max_length=50)
```

Comment:
Should this be a free text or is there a table that defines categories?

Reply:
fixed